PWGJE/Tasks/CMakeLists.txt

```diff
@@ -303,3 +299,16 @@ if(FastJet_FOUND)
                         PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::PWGJECore O2Physics::AnalysisCore
                         COMPONENT_NAME Analysis)
 endif()
+
```

Comment:
no clue what this is, but did you test/confirm that this works? Will it work on hyperlool?